aspnetcore/fundamentals/websockets.md

```diff
@@ -77,6 +77,10 @@ The following example is from later in the `Configure` method:
 
 A WebSocket request could come in on any URL, but this sample code only accepts requests for `/ws`.
 
+To keep the `Startup` clutter-free, a similar approach can be taken in an action method:
```

Comment:
```suggestion
To keep the `Startup` clutter-free, a similar approach can be taken in a controller method:
```

>To keep the `Startup` clutter-free

I don't know if this is needed. It implies that controllers are the only way to remove code from startup which is wrong.

Reply:
```suggestion
A similar approach can be taken in an controller method:
```